### PR TITLE
SISRP-8153 - Fluid - add colors red (#ff3037) and green (#0d863d) to …

### PR DESCRIPTION
**Pull request by marcpechaitis**
…css; create class to style errors with input values

https://jira.berkeley.edu/browse/SISRP-8153

### CODE REVIEW

**Comment by marcpechaitis** [comment]
@christianv Please review

File: src/css/cs/items/_kitchensink.scss [hunk 1]
```diff
@@ -312,6 +313,21 @@
     }
   }
 
+  .uc-input-error {
+    > .ps_box-control input[type="text"] {
+      background-color: $uc-color-red;
+      background-image: none;
```
Comment (christianvuerings) [inline src/css/cs/items/_kitchensink.scss]: @marcpechaitis can we remove it?

File: src/css/cs/items/_kitchensink.scss [hunk 2]
```diff
@@ -312,6 +313,21 @@
...
+    }
+  }
+
+  .psc_error {
+    @media all {
```
Comment (christianvuerings) [inline src/css/cs/items/_kitchensink.scss]: @marcpechaitis can we try it without the `@media all`?

**Comment by marcpechaitis** [comment]
@christianv r4r

**Pull request merged**